packages/datadog-plugin-mysql/src/index.js

```diff
@@ -22,6 +22,12 @@ class MySQLPlugin extends DatabasePlugin {
         'out.port': dbConfig.port
       }
     })
+    if (this.config.dbmPropagationMode !== 'disabled') {
+      if (sqlStatement[0].sql !== undefined) {
```

Comment:
This normalization should be done on the subscriber side so that we don't have to send the whole arguments object through.

Reply:
@bengl the issue I found is that when doing this on the subscriber side I end up just passing along the string, and not the reference to the place in the object I need to modify.

Reply:
This was fixed with @tlhunter and @bengl